packages/api/src/useAPI.ts

```diff
@@ -12,6 +12,13 @@ import APIContext from './APIContext';
 
 const useAPI = () => {
     const api = useContext(APIContext);
+    const [isReady, setIsReady] = useState<boolean>(api?.get()?.connection?.readyState === WebSocket.OPEN);
```

Comment:
```suggestion
    const [isReady, setIsReady] = useState<boolean>(api?.connection?.readyState === WebSocket.OPEN);
```

Does this not work? Without the `get` method?

Reply:
@markwylde-deriv as I mentioned above without `get` it will work with standalone flag, but will not work for legacy WS